packages/odyssey-react-mui/package.json

```diff
@@ -96,6 +98,7 @@
     "@types/jest-axe": "^3.5.1",
     "@types/react": "^17.0.30",
     "@types/react-dom": "^17.0.5",
+    "@types/react-window": "^1",
```

Comment:
That's weird. The `@types` library doesn't have a matching version?

Reply:
This has been resolved. I don't know why this appeared here, since I never explicitly added the types package until you pointed this out. Should be good now